db/migrate/201512311928_delete_mastery_on_user.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteMasteryOnUser < ActiveRecord::Migration
+  def change
+    remove_column :users, :mastery
+  end
+end
```

Comment:
Even though I can't imagine ever having to roll back, I'd like to have separate `up` and `down` methods here, where the `down` provides the proper data type etc.